lib/likeno/base.rb

```diff
@@ -0,0 +1,69 @@
+module Likeno
```

Comment:
This file contains a lot of duplication with [Likeno::Entity](https://github.com/mezuro/likeno/blob/request_methods_module/lib/likeno/entity.rb). Is that right?